pkg/metrics/kubernetes_client.go

```diff
@@ -109,14 +110,14 @@ func init() {
 		Name:        "k8s_client_cache_watch_duration_seconds",
 		Measure:     cacheWatchesLatencyStats,
 		Description: "Duration of watches on the Kubernetes API.",
-		Aggregation: view.Distribution(0),
+		Aggregation: view.Distribution(distributionSeconds...),
 	}))
 
 	runtime.Must(view.Register(&view.View{
 		Name:        "k8s_client_cache_watch_events",
 		Measure:     cacheItemsInWatchesCountStats,
 		Description: "Number of items in watches on the Kubernetes API.",
-		Aggregation: view.Distribution(0),
+		Aggregation: view.Distribution(0.00001),
```

Comment:
Same here. you should use the same kind of buckets. It work for our dashboard to not have bucket cause I do average mostly but if you want 99th % you can't without proper buckets.